src/main/kotlin/no/nav/syfo/brev/behandler/BehandlerVarselService.kt

```diff
@@ -43,22 +43,56 @@ class BehandlerVarselService(
     }
 
     fun opprettVarselSvar(
-        innkallingDialogmoteSvar: InnkallingDialogmoteSvar,
+        varseltype: MotedeltakerVarselType,
+        svarType: DialogmoteSvarType,
+        svarTekst: String?,
         conversationRef: UUID?,
-        parentRef: UUID?,
+        parentRef: UUID?
+    ) {
+        log.info("Received svar $svarType på varsel $varseltype with conversationRef $conversationRef and parentRef $parentRef")
+        when (varseltype) {
+            MotedeltakerVarselType.INNKALT -> opprettInnkallingVarselSvar(
+                type = svarType,
+                tekst = svarTekst,
+                conversationRef = conversationRef
+            )
+            MotedeltakerVarselType.NYTT_TID_STED -> opprettEndringVarselSvar(
+                type = svarType,
+                tekst = svarTekst,
+                conversationRef = conversationRef,
+                parentRef = parentRef
+            )
+            else -> throw IllegalArgumentException("Could not create svar for varsel $varseltype")
+        }
+    }
+
+    private fun opprettInnkallingVarselSvar(
+        type: DialogmoteSvarType,
+        tekst: String?,
+        conversationRef: UUID?
     ) {
-        when (innkallingDialogmoteSvar.foresporselType) {
-            ForesporselType.INNKALLING -> {
-                log.info("Received dialogmote-svar på INNKALLING with conversationRef $conversationRef and parentRef $parentRef")
-                // TODO: Finn innkalling-varsel til behandler i databasen og lagre svar på varselet
-            }
-            ForesporselType.ENDRING -> {
-                log.info("Received dialogmote svar på ENDRING with conversationRef $conversationRef and parentRef $parentRef")
-                // TODO: Finn endring-varsel til behandler i databasen og lagre svar på varselet
-            }
+        val pMotedeltakerBehandlerVarsel =
+            conversationRef?.let { database.getMotedeltakerBehandlerVarselForUuid(it) }
+        if (pMotedeltakerBehandlerVarsel?.varselType == MotedeltakerVarselType.INNKALT) {
+            database.createMotedeltakerBehandlerVarselSvar(
+                motedeltakerBehandlerVarselId = pMotedeltakerBehandlerVarsel.id,
+                type = type,
+                tekst = tekst,
+            )
+        } else {
+            log.warn("Could not find MotedeltakerBehandlerVarsel of type ${MotedeltakerVarselType.INNKALT.name} for dialogmote-svar på INNKALLING")
```

Comment:
Her bør vi tenke på en fallback-løsning, men det trenger ikke være med i denne PR'en.

Reply:
Ja, tar det i senere PR. Men vi kan gjerne ta en prat om det og om logikk for å finne endrings-varsel.